### PR TITLE
Blazor 0.9 coreapp

### PR DESCRIPTION
**Pull request by jivanova**

### CODE REVIEW

File: TelerikBlazor.Server/DataAccess/NorthwindContext.cs [hunk 1]
```diff
@@ -36,7 +33,7 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
             if (!optionsBuilder.IsConfigured)
             {
                 //To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
-                optionsBuilder.UseSqlServer("Server=(LocalDb)\\MSSqlLocalDb;Database=Northwind;Trusted_Connection=True;");
+                optionsBuilder.UseSqlServer(@"Server=JIVANOVA\SQLEXPRESS;Database=Northwind;Trusted_Connection=True;");
```
Comment (ag-petrov) [inline TelerikBlazor.Server/DataAccess/NorthwindContext.cs]: Can we remove the personal cridentials? Is the readme updated to note users that they should replace this string?

File: TelerikBlazor.sln [hunk 2]
```diff
@@ -1,31 +1,43 @@
...
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TelerikBlazor.Server", "TelerikBlazor\TelerikBlazor.Server\TelerikBlazor.Server.csproj", "{DB13D0B5-39C3-455E-96DF-D34B342290BA}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TelerikBlazor.Server", "TelerikBlazor.Server\TelerikBlazor.Server.csproj", "{DDFB771E-5733-40F3-B9D4-844501100BBC}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TelerikBlazor.App", "TelerikBlazor\TelerikBlazor.App\TelerikBlazor.App.csproj", "{FA4EE663-93A7-4BFA-B92E-58406DEE8818}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TelerikBlazor.Client", "TelerikBlazor.Client\TelerikBlazor.Client.csproj", "{F1B213CF-C781-4C1A-8692-63D8E978AD3E}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TelerikBlazor.Shared", "TelerikBlazor.Shared\TelerikBlazor.Shared.csproj", "{7A77AA19-B538-491A-B701-76F9A1E5A02A}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Kendo.Blazor", "D:\work\git\kendo-blazor\Kendo.Blazor\Kendo.Blazor.csproj", "{AAE47115-963A-4A97-BB44-E12441F5CFE9}"
```
Comment (ag-petrov) [inline TelerikBlazor.sln]: This reference causes the wrong loading when opening the solution. It should be removed.

File: TelerikBlazor.Client/TelerikBlazor.Client.csproj [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Blazor" Version="0.9.0-preview3-19154-02" />
+    <PackageReference Include="Microsoft.AspNetCore.Blazor.Build" Version="0.9.0-preview3-19154-02" PrivateAssets="all" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\TelerikBlazor.Shared\TelerikBlazor.Shared.csproj" />
+    <ProjectReference Include="D:\work\git\kendo-blazor\Kendo.Blazor\Kendo.Blazor.csproj" />
```
Comment (ag-petrov) [inline TelerikBlazor.Client/TelerikBlazor.Client.csproj]: This should be the reference to the official Kendo Blazor dll.

File: TelerikBlazor.Server/DataAccess/NorthwindContext.cs (hunk 1 above)
Comment (jivanova) [reply TelerikBlazor.Server/DataAccess/NorthwindContext.cs]: it's already removed

**Pull request merged**